inspirehep/config.py

```diff
@@ -55,6 +55,7 @@
 FEATURE_FLAG_ENABLE_MERGER = False
 FEATURE_FLAG_ENABLE_UPDATE_TO_LEGACY = False
 """This feature flag will prevent to send a ``replace`` update to legacy."""
+FEATURE_FLAG_REMOVE_REFS_IN_WORKFLOW = True
```

Comment:
shouldn't we keep the same naming for flags i.e. ``FEATURE_FLAG_ENABLE_REMOVE_REFS_IN_WORKFLOW``? It might get really confusing in the future if we have different "types"